certified-connectors/AtBot Admin/apiProperties.json

```diff
@@ -4,13 +4,13 @@
     "connectionParameters": {
       "token": {
         "oAuthSettings": {
-          "clientId": "6145b6dc-1ce4-43b1-a4b5-a19b8440096a",
+          "clientId": "CLIENT ID",
```

Comment:
you'll see a better UI experience (and others that try to work on it) if you just set it to `null`.
`null` is not a valid value at deployment time, and so the error will happen right away, rather become some weird runtime issue.

Same below with the resourceUri

Reply:
Thanks for the feedback.  I've changed them to null.